irods/test/login_auth_test.py

```diff
@@ -199,18 +197,19 @@ def create_env_dirs(self):
         retval = dirs.keys()
         return retval
 
+    PAM_SCHEME_STRING = 'pam'
 
     @classmethod
     def setUpClass(cls):
         cls.admin = helpers.make_session()
+        if cls.admin.server_version > (4,3):
```

Comment:
Should this be >=?
```suggestion
        if cls.admin.server_version >= (4,3):
```

Reply:
I think they're equivalent because server_version will never report `(4,3)`.

Reply:
It won't hurt and will be arguably clearer I suppose. changing ....

Reply:
It won't hurt and will be arguably clearer I suppose. changing ....